src/scss/override/_alerts.scss

```diff
@@ -1,37 +1,73 @@
-﻿// Override
+//
+// Static Resources
+//
+$alert-icon-base: 48px;
```

Comment:
You're right. We do have spacer(foo). But that, at least for now, is for spacing between things, not the sizing of things.